x-pack/platform/plugins/shared/maps/public/classes/styles/color_palettes.ts

```diff
@@ -22,9 +21,19 @@ import { PercentilesFieldMeta } from '../../../common/descriptor_types';
 
 export const DEFAULT_HEATMAP_COLOR_RAMP_NAME = 'theclassic';
 
+/**
+ * This function implement the same darken method as the previously used tinycolor2 library but using chroma-js.
+ * Unfortunately we the two darken functions works slightly differently and to avoid any minimal color change we need to reimplement
+ * the same functionality.
+ */
+function darken(color: string): string {
```

Comment:
I think its ok if chroma darken function is used. Changing `DEFAULT_LINE_COLORS` will not impact existing maps since all styles are stored with a map when saved. 

I verified that colors generated with chroma darken functions work well with `DEFAULT_FILL_COLORS`.

Reply:
fixed with https://github.com/elastic/kibana/pull/220221/commits/a9f60083f3c2e4f4df265f8b7211652bb5060bdc